test/CMakeLists.txt

```
@@ -296,10 +300,23 @@ function(tbb_add_tbbbind_test)
         )
     endif()
 
-    set_tests_properties(${_tbbbind_test_NAME} PROPERTIES
-        PASS_REGULAR_EXPRESSION "oneTBB: TBBBIND.*UNAVAILABLE"
-        FAIL_REGULAR_EXPRESSION "Status:.*FAILURE"
-    )
+    if (TARGET HWLOC::hwloc_static)
+        # TBBBind is statically linked with HWLOC is available,
+        # so in the environment without HWLOC, it should be found.
+        set_tests_properties(${_tbbbind_test_NAME} PROPERTIES
+            PASS_REGULAR_EXPRESSION "oneTBB: TBBBIND.*tbbbind"
+            FAIL_REGULAR_EXPRESSION "Status: FAILURE!"
```

Why regular expression is omitted in this case? Should not it be similar to the one below? E.g.,:
```suggestion
            FAIL_REGULAR_EXPRESSION "Status:.*FAILURE"
```